source/process/product-manager-hiring.rst

```diff
@@ -0,0 +1,47 @@
+====================================
+Product Managers
```

Comment:
This can also be added to TOC. It could be optionally combined with https://docs.mattermost.com/process/pm-faq.html.

Reply:
Good point, added to TOC, created a new section for PM handbook which includes the FAQ and this page